### ISSUE TITLE
Workspace.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by blaize9** [Nicolab/atom-local-history#24]
Here is the other errors that show up with a fresh install of atom http://pastebin.com/raw.php?i=RuXUb6PM

Use ::getActivePaneItem() instead of the ::activePaneItem property
```
Workspace.Object.defineProperty.get (C:\Users\blaize9\AppData\Local\atom\app-0.190.0\resources\app\src\workspace.js:892:14)
Object.getCurrentBuffer (C:\Users\blaize9\.atom\packages\local-history\node_modules\atom-helpers\lib\editor.js:7:30)
```

**Comment by Nicolab** [comment]
Ok I will change, thank you

**Comment by Nicolab** [comment]
Released to local-history@v3.0.0 (compatible Atom v0.192 and higher).

Thanks :)
